quest/util/config.py

```diff
@@ -160,13 +159,13 @@ def save_settings(filename=None):
 
 def _default_config_file():
     base = get_settings()['BASE_DIR']
-    return os.path.join(base, 'quest_config.yml')
+    return os.path.join(base, './settings/quest_config.yml')
 
 
 def _default_quest_dir():
     quest_dir = os.environ.get('QUEST_DIR')
     if quest_dir is None:
-        quest_dir = appdirs.user_data_dir('quest', 'erdc')
+        quest_dir = os.path.join(expanduser("~"), 'Quest')
```

Comment:
Change this to: `quest_dir = os.path.join(os.path.expanduser('~'), 'Quest')`. (see commend above on the import statement.